### DISCUSSION TITLE
Vuetify Bate 14 'v-text-field' border prominent

### DISCUSSION THREAD

**Discussion by ghost** [vuetifyjs/vuetify#15937] [Q&A]
window size 100%
![1666250870822](https://user-images.githubusercontent.com/66768066/196884575-b8629f8d-07e9-46c1-bdfe-e47ae1304ccf.png)
![1666250864722](https://user-images.githubusercontent.com/66768066/196884619-230f8290-9eca-4912-891f-9b1716d79fea.png)
window size 110%
![1666250884164](https://user-images.githubusercontent.com/66768066/196884648-18b42589-ed9c-4a57-b0fc-2fc5dd9eae58.png)

**Comment by 1Luc1** [comment]
See the discussion about this issue here: https://github.com/vuetifyjs/vuetify/issues/15789
